src/plugins/discord/reactionWeights.js

```diff
@@ -42,10 +45,27 @@ export function reactionWeight(
     return 0;
   }
 
+  const roleMultipliedReactingMembers = weights.emojiWeights.applyAveraging
+    ? [
+        ...new Set(
+          reactions
+            .filter(({reaction}) => reaction.authorId !== message.authorId)
+            .map(({reactingMember}) => reactingMember)
+        ),
+      ].reduce(
```

Comment:
nit: I think this is easier to read.
```suggestion
    ? Array.from(new Set(
          reactions
            .filter(({reaction}) => reaction.authorId !== message.authorId)
            .map(({reactingMember}) => reactingMember)
        ))
      .reduce(
```